docs/logs/getting-started-aspnetcore/Program.cs

```diff
@@ -0,0 +1,55 @@
+// <copyright file="Program.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using OpenTelemetry.Logs;
+using OpenTelemetry.Resources;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Logging.AddOpenTelemetry(logging =>
+{
+    logging.IncludeScopes = true;
+    logging.ParseStateValues = true;
```

Comment:
@CodeBlanch Whats your thoughts on `ParseStateValues` in this getting started example? Its non-trivial to explain, and probably not required as well?

Reply:
As one anecdotal data point, I saw this and had no idea what it does or why the sample sets it. That isn't to say its wrong, but I've yet to go research what this does.

Reply:
Good point. I've removed it. Probably something we should cover in a different example (e.g. customizing the SDK).